recipes-core/disk-encryption/init

```diff
@@ -133,7 +129,7 @@ start() {
     log "Using disk: ${TARGET_DISK}"
 
     KEY=$(get_key)
-    if [ $? -ne 0 ] || [ -z "$KEY" ]; then
+    if [ -z "$KEY" ]; then
```

Comment:
if `get_key()` returns codes other than zero you need to revert the error checking here